siriuspy/siriuspy/devices/intlkctrl.py

```diff
@@ -453,60 +481,73 @@ def cmd_hutchA_photon_close(self, timeout=None):
     def cmd_beamline_open(self):
         """."""
         if not self.is_hutchA_intlk_search_done:
-            print('hutchA search is not done!')
+            print(self.error_log)
             return False
 
         if not self.is_hutchB_intlk_search_done:
-            print('hutchB search is not done!')
+            print(self.error_log)
             return False
 
         if not self.is_machine_gamma_enabled:
-            print('machine gamma signal not enabled.')
+            print(self.error_log)
             return False
 
         # check and reset EPS
         if not self.is_beamline_eps_ok:
-            print('beamline eps reset')
             self.cmd_beamline_eps_reset()
             t0 = _time.time()
             while not self.is_beamline_eps_ok:
                 if _time.time() - t0 > self.TIMEOUT_EPS_RESET:
-                    print('eps reset timeout reached!')
+                    print(self.error_log)
                     return False
                 _time.sleep(0.5)
 
         # check frontend shutter permission and open gatevalves for hutchA
         if not self.is_frontend_shutter_eps_permission_ok:
-            print('open frontend and hutchA gatevalves')
             # open frontend and hutchA gatevalves
             self.cmd_frontend_gatevalves_open()
             t0 = _time.time()
             while not self.is_frontend_gatevalves_opened:
                 if _time.time() - t0 > self.TIMEOUT_GATEVALVE:
-                    msg = 'open frontend and hutchA gatevalve timeout reached!'
-                    print(msg)
+                    print(self.error_log)
                     return False
                 _time.sleep(0.5)
 
         # check hutchA shutter permission and open gatevalves for hutchB
         if not self.is_hutchA_shutter_eps_permission_ok:
-            print('open hutchB gatevalves')
```

Comment:
this removal was on purpose?

Reply:
maybe your comment is deprecated ?

Reply:
yes, I am printing now only if command, which is executed if necessary, fails.